src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,17 @@
 import React from 'react';
+import { User } from '../../types/User';
+import './UserInfo.scss';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
+type Props = {
+  user: User | null,
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <p>
+    {` Posted by  ${user?.username}: `}
+
+    <a className="UserInfo" href={`mailto:${user?.email}`}>
+      {user?.name}
+    </a>
+  </p>
```

Comment:
And what will be here if there is no user?
Just don't render this component if user is null (check it in parent component)